shared/styles/css.js.flow

```diff
@@ -0,0 +1,514 @@
+// @flow
+type Color = null | string
+type Dimension = string | number // TODO this is originally string | number but flow currently barfs on this. unclear why
```

Comment:
this TODO still apply? it's still `string | number`

Reply:
sorry this is a leftover